content/en/blog/_posts/2018-07-18-11-ways-not-to-get-hacked.md

```diff
@@ -186,7 +186,7 @@ metadata:
   name: my-service
 spec:
   selector:
-    app: MyApp
+    app.kubernetes.io/name: MyApp
```

Comment:
We avoid changing blog articles that are more than 1 year old.

Reply:
Please note the consistency in this file is an issue, L189 `app: MyApp` vs L229 `app: myapp`

Have removed the changes regardless.